data/json/monsters/feral_humans.json

```diff
@@ -645,5 +645,37 @@
     ],
     "anger_triggers": [ "FRIEND_ATTACKED", "PLAYER_CLOSE", "STALK" ],
     "fear_triggers": [ "HURT" ]
+  },
+  {
+    "id": "mon_feral_soldier",
+    "type": "MONSTER",
+    "name": { "str": "feral soldier" },
+    "description": "Lorem ipsum.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 100,
+    "speed": 85,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "i_magenta",
+    "aggression": 40,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 2,
+    "melee_cut": 18,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_body_armor", "wps_humanoid_open_helmet" ],
+    "dodge": 4,
+    "harvest": "human",
+    "vision_night": 5,
+    "path_settings": { "max_dist": 40, "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true },
+    "death_drops": "mon_feral_soldier_death_drops",
+    "zombify_into": "mon_zombie_soldier",
+    "fungalize_into": "mon_zombie_fungus",
```

Comment:
```suggestion
```

Reply:
This entry is used for zombies, not ferals. If you want to create an infected variant to fungalize into like the ones above though, that would be good and cool

Reply:
Whoops, missed that, thanks